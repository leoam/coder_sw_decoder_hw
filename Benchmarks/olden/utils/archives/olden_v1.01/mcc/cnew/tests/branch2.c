
typedef struct this{
	int x;
} *This;

main()
{
	int y;
	This thing,true;
	true = mymalloc(sizeof(struct this));
        MIGRATE();
        MIGRPH();
	thing = mymalloc(sizeof(struct this));
	if(y)
	{thing->x = 2;}
	else thing->x = 7;
	y = thing->x;
	y = true->x;
}	
