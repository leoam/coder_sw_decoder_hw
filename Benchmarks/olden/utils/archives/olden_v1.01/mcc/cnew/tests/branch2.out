10: operands of = have illegal types `pointer to struct this' and `int'
13: operands of = have illegal types `pointer to struct this' and `int'
.seg "text"

.global _main
-----------------------
************************
-----------------------
************************
-----------------------
true 
local:true 
************************
-----------------------
true 
************************
-----------------------
true 
************************
-----------------------
true 
************************
-----------------------
true 
************************
-----------------------
true 
************************
-----------------------
true 
************************
-----------------------
thing 
true 
local:thing 
************************
-----------------------
thing 
true 
local:thing 
************************
-----------------------
thing 
true 
local:thing 
************************
-----------------------
thing 
true 
local:thing 
************************
-----------------------
thing 
true 
local:thing 
************************
-----------------------
thing 
true 
local:thing 
************************
-----------------------
thing 
true 
local:thing 
************************
-----------------------
thing 
true 
local:thing 
************************
all_work: 0
Nsucc: 1  howfar: 0
i:0 list:773c0
all_work: 0
Nsucc: 1  howfar: 0
i:0 list:77410
all_work: 0
Nsucc: 1  howfar: 0
i:0 list:778e8
all_work: 0
Nsucc: 1  howfar: 0
i:0 list:77938
all_work: 0
Nsucc: 1  howfar: 0
i:0 list:77998
all_work: 0
Nsucc: 1  howfar: 0
i:0 list:779e8
all_work: 0
Nsucc: 1  howfar: 0
i:0 list:77a48
all_work: 0
Nsucc: 1  howfar: 0
i:0 list:77a98
all_work: 0
Nsucc: 2  howfar: 0
i:0 list:77b98
all_work: 0
Nsucc: 1  howfar: 0
i:0 list:77bf8
all_work: 0
Nsucc: 1  howfar: 0
i:0 list:77c48
all_work: 0
Nsucc: 1  howfar: 0
i:0 list:77cb8
all_work: 0
Nsucc: 1  howfar: 0
i:0 list:77d08
all_work: 0
Nsucc: 0  howfar: 0
